baselines/disable-packing-test/src/v1beta1/echo_client.ts.baseline

```diff
@@ -607,6 +607,25 @@ export class EchoClient {
     this.initialize();
     return this.innerApiCalls.wait(request, options, callback);
   }
+/**
+ * Check the status of the long running operation returned by the wait() method.
+ * @param {String} name
+ *   The operation name that will be passed.
+ * @returns {Promise} - The promise which resolves to an object.
+ *   The decoded operation object has result and metadata field to get information from.
+ *
+ * @code sample:
```

Comment:
Tell me about this `@code` format.  How does that render in the generated `jsdoc` output?

Reply:
I was trying to provide a code piece, but `@example` might be a better annotation for this.